enterprise/coderd/scim.go

```diff
@@ -239,6 +239,8 @@ func (api *API) scimPostUser(rw http.ResponseWriter, r *http.Request) {
 			OrganizationID: defaultOrganization.ID,
 		},
 		LoginType: database.LoginTypeOIDC,
+		// Do not send notifications to user admins as SCIM endpoint might be called sequentially to all users.
+		SkipNotifications: true,
```

Comment:
Should we make this configurable?
For example: I might want to receive webhook notifications for every user created by SCIM.

Reply:
I think I will leave it until we have a use case with SCIM.